content/en/apps/guides/data/analytics/setup.md

```diff
@@ -0,0 +1,108 @@
+| `COUCHDB_PORT`            | `5984`                                             | Port of the CouchDB instance to sync with                                                                                                      |
+| `COUCHDB_SECURE`          | `false`                                            | Is connection to CouchDB instance secure?                                                                                                      |
+
+{{% alert title="Note" %}}
+If `CHT_PIPELINE_BRANCH_URL` is pointing to a private repo then you need to provide an access token in the url i.e. `https://<PAT>@github.com/medic/cht-pipeline.git#main`. In this example you will replace `<PAT>`  with an access token from Github. Instruction on how to generate one can be found [here](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens).
+{{% /alert %}}
+
+### Setup
+The following configurations facilitate data synchronization, transformation, and storage for local development and testing.
+
+Copy the values in `env.template` file to the `.env` file and update them accordingly to the local configuration for the different scenarios below.
+
+Install the dependencies:
+```sh
+npm install
+```
+
+#### With sample CouchDB data
+This setup involves starting Logstash, PostgreSQL, PostgREST, DBT, and CouchDB. Sample fake data is generated for CouchDB.
+
+Run the Docker containers and wait for every container to be up and running:
+```sh
+# starts logstash, postgres, postgrest, generator (for fake data), couchdb and dbt
+npm run local
+```
+
+This is equivalent to running:
+```sh
+# starts logstash, postgres, postgrest, generator (for fake data), couchdb and dbt
+docker-compose -f docker-compose.couchdb.yml -f docker-compose.postgres.yml -f docker-compose.yml up -d
+```
+
+#### With a separate CouchDB instance 
+This setup involves starting Logstash, PostgreSQL, PostgREST, and DBT. It assumes you have a CouchDB instance running, and you updated the `.env` CouchDB variables accordingly.
+
+Run the Docker containers locally and wait for every container to be up and running:
+```sh
+# starts logstash, postgres, postgrest, and dbt
+docker-compose -f docker-compose.postgres.yml -f docker-compose.yml up -d
+```
+
+#### With separate CouchDB and PostgreSQL instances
+This local setup involves starting Logstash, PostgREST, and DBT. It assumes that CouchDB and PostgreSQL instances are run separately from the Docker Compose provided with CHT Sync, and the `.env` variables were updated to match those instances details.
+
+Run the Docker containers locally and wait for every container to be up and running:
+```sh
+# starts logstash, postgrest, and dbt
+docker-compose -f docker-compose.postgrest.yml -f docker-compose.yml up -d logstash postgrest dbt
```

Comment:
We just told them what this command will do, we can simplify by removing duplicate content:

```suggestion
docker-compose -f docker-compose.postgrest.yml -f docker-compose.yml up -d logstash postgrest dbt
```